cmd/http_to_nsq/main.go

```diff
@@ -37,10 +38,12 @@ func main() {
 	topic := args["--topic"].(string)
 	addr := args["--address"].(string)
 	nsqd := args["--nsqd-tcp-address"].(string)
+	subscribeTopic := args["--subscribe-topic"].(string)
```

Comment:
I think it was probably just an oversight that we didn't use `--topic` passed in with the args. This is listening on an http enpoint and publishing to nsq so I think we can just make that tweak and call it good.